Packs/CortexResponseAndRemediation/ReleaseNotes/1_1_6.md

```diff
@@ -0,0 +1,44 @@
+
+#### Playbooks
+
+##### New: Uncommon execution of ODBCConf
+
+  This playbook handles "Uncommon execution of ODBCConf" alerts.
+
+  Playbook Stages:
+
+  Analysis:
+  During the analysis, the playbook will perform the following:
+
+  - Checks if the causality process (CGO) is signed and prevalent.
+  - Checks for the host's risk score.
+
+  If the CGO is unsigned and not prevalent or either of them plus with a high-risk score, it proceeds to remediation actions; otherwise, it continues to the investigation phase.
```

Comment:
If the CGO process is not signed and not prevalent, or if either of these conditions is met in addition to having a high-risk score, the playbook proceeds with remediation actions. Otherwise, it will continue to the investigation phase.